Dockerfile-server

```diff
@@ -1,6 +1,23 @@
 FROM docker.io/node:alpine
+
+RUN apk -U upgrade --no-cache
+
 WORKDIR /backend
+
 COPY package.json .
-RUN yarn install
-COPY . .
-CMD yarn backend
+COPY yarn.lock .
+COPY babel.config.json .
+COPY tsconfig.json .
+COPY src/ ./src
```

Comment:
Isn't it easier and less error prone to copy all and filter out in the `.dockerignore` file ?